dist/s_style

```diff
@@ -59,11 +59,11 @@ else
 
 	# Finding paired typos in the comments of different file types and excluding invalid edge cases.
 	if [ "${fname##*.}" = "py" ]; then
-		egrep '#.*\b([a-zA-Z]+)\s\1\b' $f | egrep -v -e "in-memory" -e "one-file" > $t
+		egrep '#.*\b\s([a-zA-Z]+)\ \1\b[[:space:]\.]' $f > $t
 	elif [ "${fname##*.}" = "c" ] || [ "${fname##*.}" = "h" ]; then
-		egrep '/?*.*\b([a-zA-Z]+)\s\1\b' $f | egrep -v -e "@" -e "long long"> $t
+		egrep '/?\*.* \b([a-zA-Z]+) \1\b[[:space:]\.]' $f | egrep -v -e "@" -e "long long" > $t
 	else
-		egrep '\b([a-zA-Z]+)\s\1\b' $f | egrep -v -e "@" -e "^\(" -e "long long" -e "::" -e "[^.]"  > $t
+		egrep '\b\s([a-zA-Z]+)\ \1\b[[:space:]\.]' $f | egrep -v -e "@" -e "^\(" > $t
```

Comment:
in addition to Jie's comment, this now needs a space after the word boundary - should this be optional? not saying it's wrong, it's a genuine question. it's also changed for the python-specific regex above.

Reply:
Thanks for pointing that out! I think it's alright to leave it as is because most of the instances of paired typos would occur in comments as either the middle part of the sentence followed by a whitespace or at the end of a sentence followed by a period or newline as per comment convention.

Reply:
cool, that makes sense to me.